### PR TITLE
create detail pesanan sribaduga page

### PR DESCRIPTION
**Pull request by rizfirman**
## Related
[integrasi detail daftar pesanan](https://app.clickup.com/t/9003239225/CES-6131)

## Changes
- create detail pesanan sribaduga page

## Evidence
title: create detail pesanan sribaduga page
project: Jabar Super Apps
participants: @rizfirman @marsellavaleria19 @rayfajars

### CODE REVIEW

File: components/Sribaduga/DaftarPesanan/Detail/index.vue [hunk 1]
```diff
@@ -182,22 +213,31 @@ export default {
...
+        path: '/sribaduga/daftar-pesanan',
         query: this.$route.query,
       })
     },
     formatDate,
     showViewDetailPage(namePage) {
       this.viewDetail = namePage
     },
+    showColorStatus(resStatus, statusHardCode) {
+      const getColorStatus = statusHardCode.find(
```
Comment (marsellavaleria19) [inline components/Sribaduga/DaftarPesanan/Detail/index.vue:225]: Hello @rizfirman ,`getColorStatus` is variable or not?? If variable, you can name that variable using noun. example : `colorStatus`..

File: components/Sribaduga/DaftarPesanan/Detail/index.vue [hunk 2]
```diff
@@ -182,22 +213,31 @@ export default {
...
+        (status) => status.statusCode === resStatus
+      )
+      return getColorStatus?.color
+    },
+    showLabelStatus(resStatus, statusHardCode) {
+      const getLabelStatus = statusHardCode.find(
```
Comment (marsellavaleria19) [inline components/Sribaduga/DaftarPesanan/Detail/index.vue:231]: It's same too..

**Review by rayfajars — APPROVED** [review]

File: components/Sribaduga/DaftarPesanan/Detail/index.vue (hunk 1 above)
Comment (rizfirman) [reply components/Sribaduga/DaftarPesanan/Detail/index.vue:225]: hi @marsellavaleria19. thanks for ur advice. yes that's a variable. I've fixed it.

File: components/Sribaduga/DaftarPesanan/Detail/index.vue (hunk 2 above)
Comment (rizfirman) [reply components/Sribaduga/DaftarPesanan/Detail/index.vue:231]: this too

**Review by marsellavaleria19 — APPROVED** [review]

**Pull request merged**